mlrun/utils/helpers.py

```diff
@@ -1687,6 +1687,24 @@ def validate_component_version_compatibility(
     return True
 
 
+def enrich_node_selectors(
+    project_node_selector: dict, runtime_node_selector: dict
```

Comment:
here it says `runtime_node_selector` but the log says `Enriching run`.
it should be generic so you don't need to specify it its a runtime or a run